vc_zoom/indico_vc_zoom/task.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2021 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+import datetime
+
+from indico.core import signals
+from indico.core.celery import celery
+from vc_zoom.indico_vc_zoom.api import ZoomIndicoClient
+from vc_zoom.indico_vc_zoom.util import ZoomMeetingType
+
+
+@signals.event.updated.connect
+@signals.event.contribution_updated.connect
+@signals.event.session_block_updated.connect
+def check_meetings(target, changes):
+    if 'start_dt' not in changes:
+        return
+    prev_date, new_date = changes.get('start_dt')
+    if new_date < prev_date:
+        return
```

Comment:
Shouldn't going back in time, as long as it's still in the future, be handled as well?

Reply:
At first sight, I don't see any case. Because meetings should always be aligned with the event date, if we're going to the past, the expiration should always be in the future (and never crossed).

Reply:
Sure, this isn't so much about expiry but about keeping a scheduled Zoom meeting in sync with the indico event (the zoom client reminds you a few minutes before one of your meetings starts)

Reply:
Ah well, we can do - that's much simpler. I assumed we only wanted to update when strictly necessary.